### ISSUE TITLE
Allow the server to issue commands

### ISSUE THREAD

**Issue by luke-jr** [tananaev/traccar-client-android#272]
I would like to have a way for my server to ask the client to open a SSH tunnel on demand. This could be done by simply executing a command provided by the server. It would make sense to require enabling this feature explicitly, and only allow it when encryption is enabled.

**Comment by tananaev** [comment]
Probably the easiest way to send command to the client is through HTTP response, but of course client needs to send a request with location data first.

**Comment by luke-jr** [comment]
Yes, that was exactly my thought. The primary purpose of Traccar of course is to send location. This gives it an ideal opportunity to receive such a command in the HTTP reply.
